src/main/io/serial.c

```diff
@@ -115,7 +115,7 @@ void pgResetFn_serialConfig(serialConfig_t *serialConfig)
         serialConfig->portConfigs[i].peripheral_baudrateIndex = BAUD_115200;
     }
 
-    serialConfig->portConfigs[0].functionMask = FUNCTION_MSP | FUNCTION_DEBUG_TRACE;
+    serialConfig->portConfigs[0].functionMask = FUNCTION_MSP | FUNCTION_LOG;
```

Comment:
Probably we should revert this and not set `FUNCTION_LOG` on any ports. Since the intention is to leave at least error messages always enabled and written to blackbox. Otherwise we might be sending LOG frames via MSP to callers that are not ready for it (specially MSPv1 only callers).

Reply:
Makes sense